modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -533,6 +538,16 @@ def reindex(self, axis, labels, **kwargs):
         return self.__constructor__(new_modin_frame)
 
     def reset_index(self, **kwargs):
+        allow_duplicates = kwargs.pop("allow_duplicates", None)
+        names = kwargs.pop("names", None)
+        if allow_duplicates not in (None, False) or names is not None:
```

Comment:
```suggestion
        if allow_duplicates or names:
```
?

Reply:
Disagree; the user might pass `allow_duplicates=[]` or `names=""`, and that case is _also_ not handled by the code after this block.

The intention here is "fallback to pandas _if_ we receive any non-default value for these parameters".

Reply:
> The intention here is "fallback to pandas if we receive any non-default value for these parameters".

Okay, this is more obvious.